tamr_client/dataset/dataframe.py

```diff
@@ -60,3 +64,122 @@ def upsert(
         {primary_key_name: pk, **json.loads(row)} for pk, row in serialized_records
     )
     return record.upsert(session, dataset, records, primary_key_name=primary_key_name)
+
+
+def create_dataset(
```

Comment:
Given that `dataframe.upsert` already exists, should we be consistent and name this just `create`?

```suggestion
def create(
```

Reply:
This was intended to avoid ambiguity over whether it is a dataset or a dataframe that is being created.  Given that there is no function to create a dataframe this is not really an issue but I'm not 100% sure here.

Reply:
I've convinced myself. The namespacing makes sense with `create`.  You can either create a dataset with `dataset.create` (read "dataset: create") or from a dataframe with `dataset.dataframe.create` (read: "dataset: dataframe-create").